src/OrchardCore/OrchardCore.Workflows.Abstractions/OrchardCore.Workflows.Abstractions.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\OrchardCore.DisplayManagement\OrchardCore.DisplayManagement.csproj" />
+    <ProjectReference Include="..\OrchardCore.Entities.DisplayManagement\OrchardCore.Entities.DisplayManagement.csproj" />
```

Comment:
Are you sure you want to add this dependency?

Reply:
I think so - VS added it for me when I wanted to use the As<> extension method on IEntity.

Reply:
It's in `OrchardCore.Entities` which is already referenced. Try to just remove it, and as a `using OrchardCore.Entities;`

Reply:
I checked and this is for `SectionDisplayDriver`